CarboPlanner/src/components/TextInput.tsx

```diff
@@ -0,0 +1,34 @@
+import { TextInput } from "react-native-paper"
+
+
+export const LongTextInput = (props : any) => {
+return (
+<TextInput 
+    style={{width:"100%", backgroundColor: "white"}}
+    theme={{roundness: 10}}
+    underlineColorAndroid="transparent"
+    mode='outlined'
+    label={props.label}
+    activeOutlineColor='black'
+    keyboardType={props.keyboard}
+    onChangeText={props.onChange}
+    right={<TextInput.Affix text={props.affix}/>}
+/>
+)
+}
+
+export const ShortTextInput = (props : any) => {
```

Comment:
Since you say text, I would suggest using string instead of any

```suggestion
export const ShortTextInput = (props : string) => {
```

Reply:
Semantics have been fixed, but props any stays due to custom settings on each textinput given through props